synapse/federation/federation_client.py

```diff
@@ -645,23 +659,22 @@ def _do_send_join(self, destination, pdu):
         # content.
         return resp[1]
 
-    @defer.inlineCallbacks
-    def send_invite(self, destination, room_id, event_id, pdu):
-        room_version = yield self.store.get_room_version_id(room_id)
+    async def send_invite(self, destination, room_id, event_id, pdu):
+        room_version = await self.store.get_room_version_id(room_id)
 
-        content = yield self._do_send_invite(destination, pdu, room_version)
+        content = await self._do_send_invite(destination, pdu, room_version)
```

Comment:
Why isn't `_do_send_invite` async'ed?

Reply:
because I forgot it.

Done in 6bbd890f05b59ffa33769ae15ea73bdf3c8dc908.

Reply:
> because I forgot it.

(For some reason I'm imaging you singing that to the tune of https://www.youtube.com/watch?v=D_XI_290cfw)

Reply:
Have you ever seen Billie Piper and me in the same room?

Reply:
:open_mouth: